vocabularies/voc4cat/0007847.ttl

```diff
@@ -0,0 +1,13 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007847>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007847"^^xsd:token ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000181> ;
+    skos:definition "Additive that is added to the support material or catalytically active species to tune the catalytic performance."@en ;
```

Comment:
Maybe change: "is added to the support material or catalytically active species" --> "is added in low concentration to the support or catalytically active material".